### PR TITLE
created a route for a single product

### PR DESCRIPTION
**Pull request by mpiech1813**
This is a new route that takes an id of the product and displays the details for it:
/api/products/:id

### CODE REVIEW

File: server/api/products.js [hunk 1]
```diff
@@ -3,11 +3,23 @@ const productsRouter = require('express').Router();
 
 productsRouter.get('/', async (req, res, next) => {
   try {
-    // const products = Products.findAll();
+    // const products = await Products.findAll();
```
Comment (JonathanCrider) [inline server/api/products.js:6]: should be Albums.findAll

**Review by JonathanCrider — CHANGES_REQUESTED** [review]
model Products has been changed to Albums

File: server/api/products.js (hunk 1 above)
Comment (mpiech1813) [reply server/api/products.js:6]: thank you that has been fixed

**Review by mpiech1813 — COMMENTED** [review]
thank you

**Review by JonathanCrider — APPROVED** [review]
perfect thanks!

**Pull request merged**